modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -146,6 +146,7 @@ Flags:
       --mcp-name string                   MCP name corresponding to the target machines (required)
       --must-gather-dir-path string       Must gather directory path (default "must-gather")
       --offlined-cpu-count int            Number of offlined CPUs
+      --per-pod-power-management          Enable Per Pod Power Management
```

Comment:
```suggestion
      --per-pod-power-management          Enable per-pod power management
```
See [Performance Profile Creator arguments](https://docs.openshift.com/container-platform/4.13/scalability_and_performance/cnf-create-performance-profiles.html#performance-profile-creator-arguments_cnf-create-performance-profiles) and [Save power with per-pod power management](https://www.redhat.com/architect/power-management-openshift-edge).

Reply:
I ran on a cluster 
```
podman run --rm --entrypoint performance-profile-creator registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.13 -h
A tool that automates creation of Performance Profiles

Usage:
  performance-profile-creator [flags]

Flags:
      --disable-ht                        Disable Hyperthreading
  -h, --help                              help for performance-profile-creator
      --info string                       Show cluster information; requires --must-gather-dir-path, ignore the other arguments. [Valid values: log, json] (default "log")
      --mcp-name string                   MCP name corresponding to the target machines (required)
      --must-gather-dir-path string       Must gather directory path (default "must-gather")
      --offlined-cpu-count int            Number of offlined CPUs
      --per-pod-power-management          Enable Per Pod Power Management
      --power-consumption-mode string     The power consumption mode.  [Valid values: default, low-latency, ultra-low-latency] (default "default")
      --profile-name string               Name of the performance profile to be created (default "performance")
      --reserved-cpu-count int            Number of reserved CPUs (required)
      --rt-kernel                         Enable Real Time Kernel (required)
      --split-reserved-cpus-across-numa   Split the Reserved CPUs across NUMA nodes
      --topology-manager-policy string    Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted] (default "restricted")
      --user-level-networking             Run with User level Networking(DPDK) enabled

```

Reply:
Dang it! I didn't realize it was output! Thanks!!